files/initramfs-tools/union-mount.j2

```diff
@@ -85,5 +85,12 @@ then
   mount -t tmpfs -o rw,nosuid,nodev,size=${varlogsize}M tmpfs ${rootmnt}/var/log
   [ -f ${rootmnt}/host/disk-img/var-log.ext4 ] && rm -rf ${rootmnt}/host/disk-img/var-log.ext4
 else
+  [ -f ${rootmnt}/host/disk-img/var-log.ext4 ] && fsck.ext4 -v -p ${rootmnt}/host/disk-img/var-log.ext4 2>&1 \
```

Comment:
>fsck.ext4 [](start = 50, length = 9)

Is there an option `-y` to prevent interactive prompt?

Reply:
The -p option that we use here will prevent interactive prompt.
The "-p" option as documented in fsck.ext3 and fsck.ext4 :
-p                   Automatic repair (no questions)